From 54fc1cfe6e4025e09b9e9fc1e2145158414140cd Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Marco=20Trevisan=20=28Trevi=C3=B1o=29?= <mail@3v1n0.net>
Date: Fri, 28 Jun 2019 20:14:27 +0200
Subject: [PATCH] window-x11: Add lost definition for
 TAKE_FOCUS_FALLBACK_DELAY_MS

In commit 8abd1b009 (cherry-pick to gnome-3-30 of commit f71151a5d) the
definition of TAKE_FOCUS_FALLBACK_DELAY_MS was not properly merged.

Include this back to fix the build failure.

https://gitlab.gnome.org/GNOME/mutter/merge_requests/307
(cherry picked from commit a51188f8d79e018425f2469b47c01f1ddde9227d)
---
 src/x11/window-x11.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/x11/window-x11.c b/src/x11/window-x11.c
index d86d35f61..360299c35 100644
--- a/src/x11/window-x11.c
+++ b/src/x11/window-x11.c
@@ -55,6 +55,8 @@
 #include "backends/meta-logical-monitor.h"
 #include "backends/x11/meta-backend-x11.h"
 
+#define TAKE_FOCUS_FALLBACK_DELAY_MS 250
+
 G_DEFINE_TYPE_WITH_PRIVATE (MetaWindowX11, meta_window_x11, META_TYPE_WINDOW)
 
 static void
-- 
2.20.1

